charts/keycloakx/values.yaml

```
@@ -364,6 +364,8 @@ dbchecker:
       memory: "32Mi"
 
 database:
+  # don't create secret for db password. Instead use existing k8s secret
+  # existingSecret: "my-existent-dbpass-secret"
```

```suggestion
  # existingSecret: "my-existent-dbpass-secret"
  existingSecret: ""
```
to avoid nil pointer